src/api/app/models/token/workflow.rb

```diff
@@ -10,7 +10,10 @@ def call(options)
     return unless extractor.allowed_event_and_action?
 
     scm_extractor_payload = extractor.call
-    yaml_file = Workflows::YAMLDownloader.new(scm_extractor_payload).call
+    yaml_downloader = Workflows::YAMLDownloader.new(scm_extractor_payload)
+    yaml_file = yaml_downloader.call
+    raise Token::Errors::NonExistentWorkflowsFile, yaml_downloader.errors.join('\n') if yaml_file.blank?
```

Comment:
@vpereira the exception you've created wasnt raised in my tests, since `yaml_file` is not blank when the `workflow.yaml` doesnt exist, `yaml_file` contains `["404 Not Found"]`

Reply:
It would also be nice if you would include a message so customers know what they can do.

Reply:
@hennevogel we are using the error message coming from `Down`.. Do you want a static message, like 'workflows.yml couldn't be found' ?

Reply:
@krauselukas please could you look again?

Reply:
@hennevogel please take a look again

Reply:
@vpereira okay, works now

Reply:
What would be an example of an error message coming from `Down`? It might already be good enough.

Reply:
@dmarcoux so in my tests the error message from down was the simple "404 Not Found"
with the exception class from victor it looks like this ```<status code="non_existent_workflows_file">
  <summary>404 Not Found</summary>
</status>```